index.html

```diff
@@ -27,6 +27,24 @@ <h2>Седона - небольшой городок в Аризоне,<br>
         </h2>
         <p>Рассмотрим 5 причин, по которым Седона лучше, чем Гранд Каньон!</p>
       </section>
+      <section>
+        <ol>
+          <li><h3>Настоящий городок</h3>-№1-Седона - не аттракцион для туристов, там течет своя жизнь
+            <img src="img/1-real-city.jpg" alt="Фото города" width="800" height="256">
+            <ul>
```

Comment:
Текст лучше обернуть в параграф, так как у нас кроме него есть другие элементы:

```suggestion
          <li>
            <h3>Настоящий городок</h3>
            <span>-№1-</span>
            <p>Седона - не аттракцион для туристов, там течет своя жизнь</p>
            <img src="img/1-real-city.jpg" alt="Фото города" width="800" height="256">
            <ul>
```